portfolio/index.html

```diff
@@ -4,7 +4,7 @@
     <meta charset="UTF-8" />
     <link rel="icon" type="image/svg+xml" href="/vite.svg" />
```

Comment:
If you want to update what icon is used in the tab, you can add a new icon in place of `vite.svg` in the repo and reference it here